app/models/katello/content_view_version.rb

```diff
@@ -44,6 +44,8 @@ class ContentViewVersion < Katello::Model
 
     validates_lengths_from_database
 
+    validates :minor, :uniqueness => {:scope => [:content_view_id, :major], :message => N_("Major, and content_view_id trio must be unique")}
```

Comment:
This message auto-assumes that it will start with the word "minor" since that's the key its validating. The message should be something like `", major, and content_view_id trio must be unique"` for it to print properly

Reply:
As in they must all three be unique or unique within the scope of each other? Wondering if we can find a word or two that encompasses that for clarity. Trio makes me think I am ordering an appetizer, which makes me hungry.

Reply:
this turned out to be a fixture issue, we are fixing now.

here's an appetizer trio though :sushi: :shallow_pan_of_food: :poultry_leg:

Reply:
:hamburger: